src/components/input-location/input-location.ts

```diff
@@ -113,22 +120,71 @@ export class InputLocationComponent {
     }
     else {
       // Map address to search-editor
```

Comment:
@tuxpiper If the user has only filled in one of the parameters in the address search, there will be no commas and it is not picked up on row 109 :/ That's why I added it in the else-section instead.

Reply:
Ohh such an oversight, that's right!  I removed it because it wasn't working at all for me when filling more than one field (the other if-else clause to decode as coordinates was being triggered).  So we need to have both of our new if-else clauses, yours and mine! Pushing a new commit now

Reply:
:D Teamwork!